### ISSUE TITLE
When running multiple tests and the first test fails, then all other tests also fail

### ISSUE THREAD

**Issue by Elias011** [codeforequity-at/botium-core#604]
**Describe the bug**
When running multiple tests and the first test fails, then all other tests also fail.

**To Reproduce**
- In a folder have multiple test files (We used excel format)
- Let the first test fail intentionally
- Run those tests using Botium-CLI 
 `botium-cli run mochawesome --config .\botium.json --convos Tests/`
- The first test will fail
- The other tests will also fail

**Expected behavior**
Only the first test fails and the rest succeed

**Screenshots and Log files**
[BotiumTestResultLog.txt](https://github.com/codeforequity-at/botium-core/files/5879045/BotiumTestResultLog.txt)
[BotiumVerboseLog.txt](https://github.com/codeforequity-at/botium-core/files/5879046/BotiumVerboseLog.txt)

**Botium Flavour:**
 Botium CLI

**Botium Version**
 1.0.3

**Additional context**
We ran botium-cli using the docker image on docker hub. But we could also reproduce this problem with a locally installed botium-cli. Our current workaround is running botium-cli docker image version 0.0.77, where the issue is not present.

We use Botium to test a Chatbot running on Microsoft Bot Framework using the DirectLine connector.

**Comment by codeforequity-at** [comment]
I suspect this to be the reason why test cases are failing:

```
  DIRECTLINE3_ACTIVITY_TEMPLATE: {
    from: { name: 'Botium', id: '00000000-0000-0000-0000-000000000000' },
    locale: 'nl-NL',
    channelId: 'directline'
  },
```
All test cases are using the same user id, so the conversations can overlap if there are only minor problems with the timing.

I suggest to remove the id-attribute there and let Botium generate a unique id for each test run - if this is possible for your Bot Framework implementation.

**Comment by codeforequity-at** [comment]
I cannot see any real trouble in the log files.

In the test case there seems to be a wrong ".2" suffix for the "Hoi" conversation step:

"Hoi, met de chatbot van SnelStart." expected to match "Hoi, met de chatbot van SnelStart.2"


You can try to set the capability "DIRECTLINE3_GENERATE_USERNAME" to true, at this is actually required to make Botium generate unique ids as username - let's see if this helps.

**Comment by Elias011** [comment]
We've tried your solution but didn't generate a unique id for each test file

We added the ".2" suffix in the test, to let the test intentionally fail. This is to reproduce the problem of having the first test fail, which then causes the other test running afterwards to also fail. Even though if the other tests normally succeed.

**Comment by Elias011** [comment]
We tried your solution, but it did not generate an unique id for each test file. So the problem is still present.
We added the ".2" suffix to reproduce the problem of having the first test fail, which then causes the other test running after that test to also fail.
We noticed in the logs that the second test is trying to assert a reply from the first test case (see screenshot)

Also, this behavior does not happen in version 0.077.
 
[BotiumVerboseLogWithGeneratedIdConfig.txt](https://github.com/codeforequity-at/botium-core/files/5887112/BotiumVerboseLogWithGeneratedIdConfig.txt)
[BotiumTestResultLogWithGeneratedIdConfig.txt](https://github.com/codeforequity-at/botium-core/files/5887114/BotiumTestResultLogWithGeneratedIdConfig.txt)
![WeirdTestResult](https://user-images.githubusercontent.com/29075753/106139223-765bf680-616d-11eb-8fde-23d571a41067.png)
